testdata/project-v3-with-deploy-image/controllers/busybox_controller.go

```diff
@@ -176,6 +223,16 @@ func (r *BusyboxReconciler) Reconcile(ctx context.Context, req ctrl.Request) (ct
 		return ctrl.Result{Requeue: true}, nil
 	}
 
+	// The following implementation will update the status
+	meta.SetStatusCondition(&busybox.Status.Conditions, metav1.Condition{Type: typeAvailableBusybox,
+		Status: metav1.ConditionTrue, Reason: "Reconciling",
+		Message: fmt.Sprintf("Deployment for custom resource (%s) created succefully", busybox.Name)})
```

Comment:
```suggestion
		Message: fmt.Sprintf("Deployment for custom resource (%s) with %d replicas created successfully ", busybox.Name)}) 
```

@NikhilSharmaWe let's add the number of replicas here, The size as well